app/services/current-session.js

```diff
@@ -134,7 +134,11 @@ export default class CurrentSessionService extends Service {
   }
 
   get canAccessMandaat() {
-    return this.canAccess(MODULE_ROLE.MANDATENBEHEER);
+    return this.canAccess(MODULE_ROLE.MANDATENBEHEER) && config.mandatenbeheer;
```

Comment:
Isn't `config.mandatenbeheer` always truthy unless we set it to an empty string in the docker-compose.yml file?

I _think_ this will always return `true` with the current code, instead of only when it's set to `'true'`?

```suggestion
    return this.canAccess(MODULE_ROLE.MANDATENBEHEER) && config.mandatenbeheer === 'true';
```